linkerd/proxy/detect/src/lib.rs

```diff
@@ -3,7 +3,7 @@ use linkerd2_error::Error;
 use linkerd2_io::{BoxedIo, Peek};
 use linkerd2_proxy_core as core;
 
-/// A strategy for detecting values out of a client transport.
+/// A strategy for detectin values out of a client transport.
```

Comment:
Nit: 
```suggestion
/// A strategy for detecting values out of a client transport.
```
also, didn't this get fixed in an earlier PR? perhaps a rebase is needed?